### PR TITLE
Kazaz/separate deploy scripts

### PR DESCRIPTION
**Pull request by kazazor**
## What's the purpose of your PR?

- [ ] Fixing a bug
- [ ] A new Feature
- [X] Techdebt
- [ ] General Task

## What is your PR all about?
* Separated the deploy scripts to different folders so we could support multiple chains
* Added BDX deployment script to arbitrum testnet so we could have something to test with (and because we need it). I did not add the deployment files of hardhat and migration. We'll do it after we'll deploy on the testnet.

### CODE REVIEW

File: deploy-scripts/deploy-scripts-constants.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { chainIds } from "../utils/Constants";
+import { TokenDeploymentData } from "./interfaces/deploy-scripts.interface";
+
+const deploymentScriptsFolder = "deploy-scripts";
+
+export const chainsDeployScriptsFolders = {
+  [chainIds.rsk]: [`${deploymentScriptsFolder}/rsk`],
+  [chainIds.mainnetFork]: [`${deploymentScriptsFolder}/rsk`],
```
Comment (AndrzejGisWideLab) [inline deploy-scripts/deploy-scripts-constants.ts:8]: This is confusing (rsk value under mainnetFork), a comment would be appreciated.

Or maybe even rename the directory to "root-deployment" or "main-deployment"

File: deploy-scripts/deploy-scripts-constants.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  [chainIds.arbitrumTestnet]: [`${deploymentScriptsFolder}/arbitrum-testnet`]
+};
+
+// Since we're deploying the same tokens on multiple chains, having this data in a constant file will prevent mistakes of typos between chains
+export const tokensDetails: TokenDeploymentData = {
```
Comment (AndrzejGisWideLab) [inline deploy-scripts/deploy-scripts-constants.ts:13]: Exporting from deployment scripts gives us troubles (e.g. when we want to run some experiments and remove deployments scripts). I think we should move it outside.

**Review by AndrzejGisWideLab — CHANGES_REQUESTED** [review]

File: deploy-scripts/deploy-scripts-constants.ts (hunk 1 above)
Comment (kazazor) [reply deploy-scripts/deploy-scripts-constants.ts:8]: I would add a comment there, makes sense

File: deploy-scripts/deploy-scripts-constants.ts (hunk 2 above)
Comment (kazazor) [reply deploy-scripts/deploy-scripts-constants.ts:13]: Not sure I follow. Even if you remove deployment scripts, they are in a sub directory for each chain.. so the experiment would be in these sub directories.. where do you see the problem here?

P.S:
This is different than the example you sent me as I create a structure of a main directory, this file and a sub directory for each chain

Comment (AndrzejGisWideLab) [reply deploy-scripts/deploy-scripts-constants.ts:13]: Nevermind, for some reason I was thinking it's a deployment script

File: deploy-scripts/deploy-scripts-constants.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { chainIds } from "../utils/Constants";
+import { TokenDeploymentData } from "./interfaces/deploy-scripts.interface";
+
+const deploymentScriptsFolder = "deploy-scripts";
+
+export const chainsDeployScriptsFolders = {
+  [chainIds.rsk]: [`${deploymentScriptsFolder}/rsk`],
+  // Our local development is based ont he deployment scripts of RSK, being the native chain of Blindex.
+  // The deployment files there have different conditions basked on the chain
```
Comment (AndrzejGisWideLab) [inline deploy-scripts/deploy-scripts-constants.ts:9]:
```suggestion
  // The deployment files there have different conditions based on the chain
```

**Review by AndrzejGisWideLab — APPROVED** [review]

**Pull request merged**